six/common/builtins/tagger.c

```diff
@@ -25,11 +25,6 @@ int parseArgs(PyObject *args, char **id, int *cardinality)
 
 PyObject *buildTagsList(char **tags)
 {
-    if (tags == NULL) {
-        // Py_RETURN_NONE macro increases the refcount on Py_None
-        Py_RETURN_NONE;
-    }
-
     PyObject *res = PyList_New(0);
     int i;
     for (i = 0; tags[i]; i++) {
```

Comment:
If tags is null we crash here. I would rather return `res` early.

Reply:
thanks, fixed

Reply:
➕